lua/harpoon/ui.lua

```diff
@@ -28,7 +28,12 @@ end
 local function create_window()
     log.trace("_create_window()")
     local config = harpoon.get_menu_config()
-    local width = config.width or 60
+    local width
+    if type(config.width) == "function" then
+        width = config.width() or 60
+    else
+        width = config.width or 60
+    end
```

Comment:
Same comment as the cmd-ui line above:
```suggestion
    local width = type(config.width) == "function" and config.width()
        or config.width
        or 60
```